request.go

```diff
@@ -42,27 +42,28 @@ type Requests struct {
 	StatusCode int          `json:"statuscode,omitempty"`
 }
 
+type RequestMetadata struct {
+	CreatedDate   time.Time `json:"createdDate"`
+	CreatedBy     string    `json:"createdBy"`
+	Etag          string    `json:"etag"`
+	RequestStatus RequestStatus `json:"requestStatus"`
```

Comment:
this is actually a breaking change: a request status' type was available as `Type` before, now its `PBType`